lib/stream_data.ex

```diff
@@ -1868,7 +1868,18 @@ defmodule StreamData do
     simple_term = one_of([boolean(), integer(), binary(), float(), atom(:alphanumeric), ref])
 
     tree(simple_term, fn leaf ->
-      one_of([list_of(leaf), map_of(leaf, leaf), {}, {leaf}, {leaf, leaf}, {leaf, leaf, leaf}])
+      one_of([list_of(leaf), map_of(leaf, leaf), one_to_four_element_tuple(leaf)])
+    end)
+  end
+
+  defp one_to_four_element_tuple(leaf) do
+    bind(integer(), fn integer ->
+      case rem(integer, 10) do
+        rem when rem >= 6 -> {leaf, leaf, leaf}
```

Comment:
Emit 3 element tuples more frequently. (4/10)

Reply:
Since you're binding on `integer()`, it means that the more the generation size grows, the more likely it will become to generate 3-element tuples. If you want to keep the percentages of `{}`, 1-element, 2-element, and 3-element tuples roughly constant, use `integer(0..10)` instead.